site/content/integrate/getting-started/how-should-i-integrate.md

```diff
@@ -26,6 +26,10 @@ That's not enough? You can include <a target="_blank" href="https://docs.matterm
 
 To build a richer bot integration, you can make full use of the [Mattermost REST API](/integrate/rest-api/). Everything that you see a Mattermost client doing, your integration can do too with this API.
 
+### I want to develop a cloud-based app 
+
+That's no problem. Our apps framework supports apps written in any language that supports HTTP, and work on our Mobile and Desktop clients. Apps can be hosted anywhere, on a Digital Ocean droplet, AWS EC2, or a server in your data centre. Take a look at our [Quickstart Guide](https://developers.mattermost.com/integrate/apps/quick-start-go/).
```

Comment:
Slash command and webhooks also use HTTP as the transport protocol and can to hosted anywhere. It point is that Apps provide a _framework_ with utils like a KV store, bot account creation  and OAuth permission scopes and -most notable- rich UX entry points.